engine/src/main/java/com/redhat/cloud/notifications/recipients/RecipientResolver.java

```diff
@@ -47,6 +60,8 @@ private Set<User> recipientUsers(String accountId, RecipientSettings request, Se
             users = filterUsers(users, subscribers);
         }
 
+        updateUsersUsedGauge(users.size());
```

Comment:
could we add the accountId somehow? I would like to compare how many we fetched on particular accounts. IIRC the rbac~ metrics do that too?

Reply:
hmm.. that's actually not really comparable I would say.

comparing the new metric from this PR with this one is more fair IMO: https://github.com/RedHatInsights/notifications-backend/blob/master/engine/src/main/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProvider.java#L95

So could we maybe leave it as it is and see if it makes sense?
Otherwise I would have to add a timer.start and timer.stop probably which we would not use.

Reply:
Thanks :) otherwise it would have become a bit hacky. Let's give it a try and if it doesnt do the job, we can still change it.